### ISSUE TITLE
GPG error messages has wrong encoding in GitHub Desktop

### ISSUE THREAD

**Issue by leofun01** [desktop/desktop#7858]
Some localized error messages shows like this:
```
������� �?� �� �஡� �?����㢠���: �?� ᪠ᮢ���
�?� ᪠ᮢ���
```

### 0. What I use:
- git 2.18.0
- gnupg v2.2.16
- OS: Windows 7 (on 8, 8.1, 10 result is the same)

### 1. Configure git to use gpg:
```git
git config --global commit.gpgsign true
git config --global gpg.program C:\path\GnuPG\bin\gpg.exe
```

### 2. git commands:
```git
git add --all
git commit -a -m "Signed commit"
```
( Cancel password input ).

#### git message (uk):
gpg: помилка під час спроби підписування: Дію скасовано
gpg: signing failed: Дію скасовано
error: gpg failed to sign the data
fatal: failed to write commit object

### 3. GitHub Desktop actions:
Click on "Commit to master" (or any other branch)
( Cancel password input ).

#### GitHub Desktop message:
Error
Commit failed - exit code 128 received, with output: 'gpg: ������� �?� �� �஡� �?����㢠���: �?� ᪠ᮢ���
gpg: signing failed: �?� ᪠ᮢ���
error: gpg failed to sign the data
fatal: failed to write commit object'

**Comment by niik** [comment]
Hi @leofun01, given that you're unable to sign your commits when using the command line I'm afraid there's not much I can do to help you here. GitHub Desktop itself doesn't yet support GPG signing although some users have reported that they've been able to successfully configure their systems to sign their commits anyway, see https://github.com/desktop/desktop/issues/78. If you're able to get the command line version of Git working with GPG signing it's possible you could follow the instructions in that issue to also get it working with GitHub Desktop.

I'm sorry that I wasn't able to provide a better answer for you but I do want to thank you for submitting this issue anyway, if/when we start investigating adding support for signing commits information like this will be very valuable to us.

I'm going to close this issue now but if you have any more context to add please do so.

**Comment by leofun01** [comment]
@niik, thank you for the quick reply, but you misunderstood me.
I have **no problems with signed commits and no problems with GPG**. I have already configured GPG and successfully used signed commits in my repositories.
But today I wanted to try what would happen if I did not enter the password. Of course, I was expecting to get an error. But this error message was displayed incorrectly.
I hope you get the purpose of my issue.
